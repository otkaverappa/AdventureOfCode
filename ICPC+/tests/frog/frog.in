35
8 3
.XX.X.X.
3 50
...
8 1
...XX...
10 4
..XXXX.XX.
2 20
..
2 0
..
21 1
..XX..XX..XX..XX..XX.
21 2
..XX..XX..XX..XX..XX.
21 3
..XX..XX..XX..XX..XX.
21 4
..XX..XX..XX..XX..XX.
21 5
..XX..XX..XX..XX..XX.
21 6
..XX..XX..XX..XX..XX.
50 4
.XXXXX..XXXXX..XXXXX..XXXXX..XXXXX..XXXXX..XXXXX..
50 5
.XXXXX..XXXXX..XXXXX..XXXXX..XXXXX..XXXXX..XXXXX..
50 10
.XXXXX..XXXXX..XXXXX..XXXXX..XXXXX..XXXXX..XXXXX..
50 20
.XXXXX..XXXXX..XXXXX..XXXXX..XXXXX..XXXXX..XXXXX..
50 30
.XXXXX..XXXXX..XXXXX..XXXXX..XXXXX..XXXXX..XXXXX..
50 0
..................................................
50 10
..................................................
50 20
..................................................
50 30
..................................................
50 40
..................................................
50 50
..................................................
50 40
.XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX.
50 48
.XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX.
50 1
.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X..
50 2
.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X..
50 3
.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X..
50 4
.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X..
50 5
.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X..
50 10
.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X..
50 20
.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X..
50 30
.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X..
50 40
.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X..
50 50
.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X..

