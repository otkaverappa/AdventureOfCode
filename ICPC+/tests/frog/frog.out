Day #1
8 3
.XX.X.X.
2

Day #2
3 50
...
1

Day #3
8 1
...XX...
0

Day #4
10 4
..XXXX.XX.
3

Day #5
2 20
..
1

Day #6
2 0
..
1

Day #7
21 1
..XX..XX..XX..XX..XX.
0

Day #8
21 2
..XX..XX..XX..XX..XX.
10

Day #9
21 3
..XX..XX..XX..XX..XX.
5

Day #10
21 4
..XX..XX..XX..XX..XX.
5

Day #11
21 5
..XX..XX..XX..XX..XX.
5

Day #12
21 6
..XX..XX..XX..XX..XX.
4

Day #13
50 4
.XXXXX..XXXXX..XXXXX..XXXXX..XXXXX..XXXXX..XXXXX..
0

Day #14
50 5
.XXXXX..XXXXX..XXXXX..XXXXX..XXXXX..XXXXX..XXXXX..
14

Day #15
50 10
.XXXXX..XXXXX..XXXXX..XXXXX..XXXXX..XXXXX..XXXXX..
7

Day #16
50 20
.XXXXX..XXXXX..XXXXX..XXXXX..XXXXX..XXXXX..XXXXX..
3

Day #17
50 30
.XXXXX..XXXXX..XXXXX..XXXXX..XXXXX..XXXXX..XXXXX..
2

Day #18
50 0
..................................................
49

Day #19
50 10
..................................................
5

Day #20
50 20
..................................................
3

Day #21
50 30
..................................................
2

Day #22
50 40
..................................................
2

Day #23
50 50
..................................................
1

Day #24
50 40
.XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX.
0

Day #25
50 48
.XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX.
1

Day #26
50 1
.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X..
25

Day #27
50 2
.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X..
24

Day #28
50 3
.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X..
13

Day #29
50 4
.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X..
12

Day #30
50 5
.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X..
9

Day #31
50 10
.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X..
5

Day #32
50 20
.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X..
3

Day #33
50 30
.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X..
2

Day #34
50 40
.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X..
2

Day #35
50 50
.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X.X..
1

